codespell_lib/data/dictionary.txt

```diff
@@ -5216,6 +5216,7 @@ certficiation->certification
 certficiations->certifications
 certfied->certified
 certfy->certify
+certi->certificate
```

Comment:
Do you mean certify, or suggest both, but this is a huge leap to auto-correct (due to only one suggestion) to certificate.

Reply:
Just done, see:
https://github.com/codespell-project/codespell/blob/f78823b28c2e8f7b7b3b622402954a784aa98548/codespell_lib/data/dictionary.txt#L5219